package.json

```diff
@@ -4,14 +4,15 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "build": "rm -rf node_modules && yarn install --production=true && tsc && zip -r --exclude=*terraform* ./grader.zip index.js node_modules/ && yarn install --production=false",
+    "build": "rm -rf node_modules && rm -rf build && yarn install --production=true && tsc && cd build && zip -r --exclude=*terraform* ../dist/grader.zip index.js node_modules/ && cd .. && yarn install --production=false",
     "typecheck": "tsc --noEmit",
-    "test": "TIMEOUT=1000 jest",
-    "test-coveralls": "TIMEOUT=1000 jest --coverage --coverageReporters=text-lcov | coveralls"
+    "test": "TIMEOUT=1000 jest --env=node",
+    "test-coveralls": "TIMEOUT=1000 jest --env=node --coverage --coverageReporters=text-lcov | coveralls"
   },
   "dependencies": {
     "@types/node": "^10.5.2",
     "js-slang": "0.1.4",
+    "lodash": "4.17.10",
```

Comment:
I can't find lodash being used in this change

Reply:
It's in `src/graphics/rune_library.js`, specifically in the code that cleans up the flattened image data (`__compile`).

Reply:
Not sure what's the magnitude change in size, but if you're using one function, individual lodash function imports are available

Reply:
Here's the exact code:

```
function __compile(shape) {
  var shape_list = generateFlattenedShapeList(shape);
  var cleaned_shape_list = shape_list.map(function (obj) {
      var shape = obj.shape;
      var instanceArray = _.mapValues(obj.instanceArray, function (value) {
          return parseFloat(value.toFixed(3));
      });
      return {
          shape: shape,
          instanceArray: instanceArray
      };
  });
  return cleaned_shape_list;
}

function __are_pictures_equal(shape0, shape1) {
    var compiled_shape0 = __compile(shape0);
    var compiled_shape1 = __compile(shape1);
    return _.isEqual(compiled_shape0, compiled_shape1);
}
```

So it's more than one function, and anyway this library is pretty general and useful just about everywhere.